src/inc/quic_platform.h

```diff
@@ -487,6 +493,15 @@ CxPlatGetTestCertificate(
     _When_(CredType == QUIC_CREDENTIAL_TYPE_CERTIFICATE_HASH_STORE, _Out_)
     _When_(CredType != QUIC_CREDENTIAL_TYPE_CERTIFICATE_HASH_STORE, _Reserved_)
         QUIC_CERTIFICATE_HASH_STORE* CertHashStore,
+    _When_(CredType == QUIC_CREDENTIAL_TYPE_CERTIFICATE_FILE, _Out_)
+    _When_(CredType != QUIC_CREDENTIAL_TYPE_CERTIFICATE_FILE, _Reserved_)
+        QUIC_CERTIFICATE_FILE* CertFile,
+    _When_(CredType == QUIC_CREDENTIAL_TYPE_CERTIFICATE_FILE_PROTECTED, _Out_)
+    _When_(CredType != QUIC_CREDENTIAL_TYPE_CERTIFICATE_FILE_PROTECTED, _Reserved_)
+        QUIC_CERTIFICATE_FILE_PROTECTED* CertFileProtected,
+    _When_(CredType == QUIC_CREDENTIAL_TYPE_CERTIFICATE_PKCS12, _Out_)
+    _When_(CredType != QUIC_CREDENTIAL_TYPE_CERTIFICATE_PKCS12, _Reserved_)
```

Comment:
The number of args is getting quite large here. I am thinking a struct that contains a union of what's necessary might be better.

Reply:
I'd prefer to keep callers simpler by not requiring a separate struct. If we add another credential type, I'm open to discussing the change then.

Reply:
Ok.